Source/Core/Common/x64Emitter.cpp

```diff
@@ -120,6 +120,16 @@ void XEmitter::ReserveCodeSpace(int bytes)
     *code++ = 0xCC;
 }
 
+const u8* XEmitter::AlignCodeTo(size_t alignment)
```

Comment:
would it be possible to implement the other alignment functions in terms of this one? (like AlignCode4, etc)